### ISSUE TITLE
Electron?

### ISSUE THREAD

**Issue by zbuster05** [Darrow8/status-app#2]
It sounds like something that could also benefit from a desktop version. Maybe try to work in electron support?

**Comment by Darrow8** [comment]
Yeah, I am planning on adding electron later. Thanks for the suggestion.
